Source/Scene/Particle.js

```diff
@@ -134,21 +129,38 @@ define([
             }
         },
         /**
-         * The dimensions of the particle in pixels. This has been deprecated. Use {@link Particle#imageSize} instead.
+         * Gets and sets the width of {@link Particle#imageSize}
          * @memberof Particle.prototype
-         * @type {Cartesian2}
-         * @default new Cartesian(1.0, 1.0)
-         * @deprecated
+         * @type {Number}
+         */
+        width : {
```

Comment:
@ggetz do you think we should remove these width and height properties?  They don't seem particularly useful since users can just access `this.imageSize` directly

Reply:
I'm fine with that, we'll just need to make sure it's updated in the tutorial if it was used there.

Reply:
They shouldn't be since they were added for the first time in the PR, but I'll double check